components/validation/src/main/java/org/wso2/carbon/identity/x509Certificate/validation/CertificateValidationUtil.java

```diff
@@ -154,7 +173,7 @@ public static void addDefaultValidationConfigInRegistry(String tenantDomain) {
                     OMElement childElement = (OMElement) iterator.next();
                     if (isValidatorConfigProperty(childElement)) {
                         validatorChildElement = childElement;
-                        addDefaultValidatorConfig(childElement, tenantDomain);
+                        addDefaultValidatorConfigToConfigStore(childElement, tenantDomain);
```

Comment:
IMO, storing data in a config with a method name `addDefaultValidationConfigInRegistry` can be misleading. Shall we update the javadoc?